main.rb

```diff
@@ -0,0 +1,103 @@
+require_relative 'app'
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+
+def display_menu
+  puts 'What would you like to do?'
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create a person'
+  puts '4. Create a book'
+  puts '5. Create a rental'
+  puts '6. List all rentals for a given person id'
+  puts '0. Quit'
+end
+
+def list_all_books(app)
+  app.list_books.each do |book|
+    puts "#{book.title} by #{book.author}"
+  end
+end
+
+def list_all_people(app)
+  app.list_people.each do |person|
+    puts "#{person.name} (#{person.class})"
+  end
+end
+
+def create_person(app)
+  puts "Enter the person's name:"
+  name = gets.chomp
+  puts "Enter the person's age:"
+  age = gets.chomp.to_i
+  puts "Enter the person's type (student or teacher):"
+  type = gets.chomp.downcase
+  if type == 'student'
+    puts "Enter the student's grade level:"
+    grade_level = gets.chomp
+    app.create_student(name, age, grade_level)
+    puts 'Student created successfully'
+  elsif type == 'teacher'
+    puts "Enter the teacher's specialty:"
+    specialty = gets.chomp
+    app.create_teacher(name, age, specialty)
+    puts 'Teacher created successfully'
+  else
+    puts 'Invalid person type'
+  end
+end
+
+def create_book(app)
+  puts 'Enter the book title:'
+  title = gets.chomp
+  puts 'Enter the book author:'
+  author = gets.chomp
+  puts 'Enter the book isbn:'
+  isbn = gets.chomp
+  app.create_book(title, author, isbn)
```

Comment:
- [ ] Please remove 'isbn' as we only need the title and the author from the user to stick with the below image from the video presentation, print the inputs on the same line and return success message after created a book.
![image](https://user-images.githubusercontent.com/91486791/232099817-b1ff389d-1fcf-49ac-8b4c-2c7dc5edfe6a.png)
![image](https://user-images.githubusercontent.com/91486791/232100018-cb00cc55-6665-4e39-b6ad-dfa04fc341e8.png)